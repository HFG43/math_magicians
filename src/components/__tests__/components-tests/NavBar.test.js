import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import NavBar from '../../navbar';

it('Renders <NavBar /> correctly', () => {
  const navBar = renderer
    .create(
      <MemoryRouter>
        <NavBar />
      </MemoryRouter>,
    )
    .toJSON();
  expect(navBar).toMatchSnapshot();
});
