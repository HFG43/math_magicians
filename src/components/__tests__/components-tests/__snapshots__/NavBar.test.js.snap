// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <NavBar /> correctly 1`] = `
<div
  className="navbar"
>
  <span
    className="navbar_title"
  >
    Math Magicians
  </span>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      |
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      |
    </li>
    <li>
      <a
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</div>
`;
